src/libs/events.ts

```diff
@@ -124,6 +124,12 @@ const eventsList = {
   workflowsTabView: 'workflowsApp:tab:view',
   workspaceClone: 'workspace:clone',
   workspaceCreate: 'workspace:create',
+  workspaceDashboardSectionToggle: 'workspace:dashboard:toggleSection',
+  workspaceDashboardAddTag: 'workspace:dashboard:addTag',
+  workspaceDashboardDeleteTag: 'workspace:dashboard:deleteTag',
+  workspaceDashboardEditDescription: 'workspace:dashboard:editDescription',
+  workspaceDashboardSaveDescription: 'workspace:dashboard:saveDescription',
+  workspaceDashboardBucketRequesterPays: 'workspace:dashboard:bucketLocationRequesterPays',
```

Comment:
Nit: Why is the event `bucketLocationRequesterPays`? Requester pays isn't an aspect of the bucket location.

Reply:
It's because this control is only shown in the case of a requester pays workspace, where the user cannot see the bucket location until they link a workspace they can bill to. This is the control beside the bucket location that allows the linking.

![image](https://github.com/DataBiosphere/terra-ui/assets/484484/a65451b0-240e-49eb-8f0e-61916d1929e9)